endpoints/openrtb2/sample-requests/valid-native/asset-with-no-id.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Can we please add some more test cases like for example, one with multiple assets with unique asset IDs, one with multiple assets but with non-unique asset IDs, one with multiple assets with unique asset IDs but one of the assets having asset ID `0`.